src/analog/m2kanalogin_impl.hpp

```diff
@@ -83,6 +83,7 @@ class M2kAnalogInImpl : public M2kAnalogIn
 
 	void setAdcCalibOffset(ANALOG_IN_CHANNEL channel, int calib_offset);
 	void setAdcCalibOffset(ANALOG_IN_CHANNEL channel, int calib_offset, int vert_offset);
+	void setAdcCalibOffset(ANALOG_IN_CHANNEL channel, int calib_offset, double vert_offset);
```

Comment:
should we add a comment to specify that one function is using the offset in Volts and the other one in Raw values? 
maybe it will help for further development

Reply:
I added some comments to clarify the usage of each function.